test/nightly/appSettings.test.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { WebSiteManagementModels as Models } from 'azure-arm-website';
+import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
+import * as vscode from 'vscode';
+import { getRandomHexString } from '../../extension.bundle';
+import { longRunningTestsEnabled, testUserInput } from '../global.test';
+import { resourceGroupsToDelete, webSiteClient } from './global.resource.test';
+
+suite('Application settings', async function (this: ISuiteCallbackContext): Promise<void> {
```

Comment:
I think test is worth putting in the webAppActions file.  Creating and deleting a resource kind of takes a while and there's really no reason to create another app just to test app settings.

In fact, I think it'd be cool if we could find a way to make it alternate between testing it on a Linux/Windows app service.  Maybe it could be something like today's date % 2 to determine whether or not to test it for Windows or Linux?  I don't know, there's probably a better way to do that 😅